firewood/src/merkle/stream.rs

```diff
@@ -396,6 +396,140 @@ impl<'a, S: ShaleStore<Node> + Send + Sync, T> Stream for MerkleKeyValueStream<'
     }
 }
 
+enum PathIteratorState<'a> {
+    Iterating {
+        /// The key, as nibbles, of the node at [address], without the
```

Comment:
I think you want this
```suggestion
        /// The key, as nibbles, of the node at `address`, without the
```
Or maybe this:
```suggestion
        /// The key, as nibbles, of the node at [Self::Iterating::address], without the
```
or maybe you need to specify the whole name
```suggestion
        /// The key, as nibbles, of the node at [PathIteratorState::Iterating::address], without the
```
I can't remember, you'll have to play with it.